languages/fr/assess-api.json

```diff
@@ -26,37 +26,37 @@
     "retry": "Réessayer",
     "save": "Enregistrer",
     "saveButtonLabel": "Quitter", 
-    "saveError": "Erreur lors de l/'enregistrement.",
+    "saveError": "Erreur lors de l'enregistrement.",
     "saveInProgress": "Enregistrement...",
     "savingInProgress": "Enregistrement en cours.",
-    "saving": "l/'enregistrement",
-    "saved": "enregsitrée",
-    "saveTestSuccessfully": "Votre activité a bien été enregistrée",
-    "saveTestSuccessfullyPrompt": "Souhaitez-vous continuer l/'{{test}}, ou quitter ?",
-    "resume": "reprendre",
+    "saving": "l'enregistrement",
+    "saved": "enregistrée",
+    "saveTestSuccessfully": "Votre {{test}} a bien été enregistrée",
```

Comment:
@shtrom, if we use {{test}} here, don't we need to use `enregistré(e)`?

Reply:
I was just ranting at Edwin about it, but missed that case. Cheers!